integration/mock-backend-server/src/main/java/org/wso2/choreo/connect/mockbackend/ResponseConstants.java

```diff
@@ -38,6 +38,9 @@ public class ResponseConstants {
     public static final String RESPONSE_VALID_JWT_TRANSFORMER = "{\"id\":3, \"category\":{\"id\":1, \"name\":\"John Doe\"}, " +
             "\"name\":\"shre\", \"photoUrls\":[\"ArrayItem1\"], \"tags\":[{\"id\":1, \"name\":\"TfNSW\"}], " +
             "\"status\":\"hello\"}";
+    public static final String RESPONSE_VALID_JWT_TRANSFORMER2 = "{\"id\":4, \"category\":{\"id\":1, \"name\":\"John Doe2\"}, " +
```

Comment:
Why did it have to be `RESPONSE_VALID_JWT_TRANSFORMER2`? Was it not related to rate limiting?